src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -36,18 +36,6 @@
       We also must reference System.Diagnostics.Debug due to this project reference and issue
       https://github.com/dotnet/corefx/issues/2817.
```

Comment:
Can you please also remove this comment and resolve issue #2519? I think issue #2817 should remain open as it I believe it applies to other libraries as well.